### PR TITLE
upgrade url validator

### PR DESCRIPTION
**Pull request by marc0der**

### CODE REVIEW

File: .github/workflows/run.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Run migration
+on: pull_request
```
Comment (eddumelendez) [inline .github/workflows/run.yml:2]:
```suggestion
on: pull_request_target
```

**Review by eddumelendez — APPROVED** [review]
run.yml already exists, isn't it? wondering why another is needed.

**Comment by marc0der** [comment]
I think I must have added that in error @eddumelendez, I'll fix the PR.

**Comment by marc0der** [comment]
That should be fixed now @eddumelendez

**Review by eddumelendez — APPROVED** [review]

**Pull request merged**